actors/common/src/main/java/org/sunbird/learner/actors/syncjobmanager/EsSyncBackgroundActor.java

```diff
@@ -40,45 +38,47 @@
   private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
 
   @Override
-  public void onReceive(Request request) throws Throwable {    
+  public void onReceive(Request request) throws Throwable {
     String operation = request.getOperation();
 
     if (operation.equalsIgnoreCase(ActorOperations.BACKGROUND_SYNC.getValue())) {
```

Comment:
operation.equalsIgnoreCase(ActorOperations.BACKGROUND_SYNC.getValue()) reverse the condition to avoid NPE